source/globalCommands.py

```diff
@@ -2286,6 +2286,21 @@ def script_recognizeWithUwpOcr(self, gesture):
 	# Translators: Describes a command.
 	script_recognizeWithUwpOcr.__doc__ = _("Recognizes the content of the current navigator object with Windows 10 OCR")
 
+	def script_toggleReportCLDR(self,gesture):

```

Comment:
Please use the script decorator to set script metadata. `script_review_currentSymbol` is a good example here.